scripts/commerce.js

```diff
@@ -575,10 +575,14 @@ export async function commerceEndpointWithQueryParams() {
  */
 export function getSkuFromUrl() {
   const path = window.location.pathname;
-  const result = path.match(/\/products\/[\w|-]+\/([\w|-]+)$/);
+  const result = path.match(/\/(?:products|products-ssg)\/[\w|-]+\/([\w|-]+)(?:\.html)?$/);
```

Comment:
Let's disable `products-ssg` route here. As I understand it, you can have overlay _and_ folder mapping, which means there is always a fallback such as if a product has not been synced to the overlay yet, but exists in the API response.

```suggestion
  const result = path.match(/\/products\/[\w|-]+\/([\w|-]+)$/);
```